app/core/Tracker2/index.js

```diff
@@ -0,0 +1,78 @@
+import SegmentTracker from './SegmentTracker'
+import CookieConsentTracker from './CookieConsentTracker'
+import LegacyTracker from './LegacyTracker'
+import BaseTracker from './BaseTracker'
+import GoogleAnalyticsTracker from './GoogleAnalyticsTracker'
+
+/**
+ * Top level application tracker that handles sub tracker initialization and
+ * event routing.
+ */
+export default class Tracker2 extends BaseTracker {
+  constructor (store) {
+    super()
+
+    this.store = store
+
+    // TODO consent status needs to be propagated to other trackers
```

Comment:
Is this propagation handled below in the `LegacyTracker`?

Reply:
It's handled in legacy tracker right now but we'll need to handle it for the other trackers as well.  Right now it's just handled in legacy tracker because that is the only place doing any meaningful tracking.